venafi/provider.go

```diff
@@ -20,67 +26,116 @@ const (
 	messageVenafiConfigFailed     = "Failed to build config for Venafi issuer: "
 	messageUseDevMode             = "Using dev mode to issue certificate"
 	messageUseVaas                = "Using VaaS to issue certificate"
+	messageUseTLSPDC              = "Using Platform TLSPDC with url %s to issue certificate"
+
+	utilityName     = "HashiCorp Terraform"
+	defaultClientID = "hashicorp-terraform-by-venafi"
 
-	utilityName = "HashiCorp Terraform"
+	// Environment variables for Provider attributes
+	envVenafiURL            = "VENAFI_URL"
+	envVenafiZone           = "VENAFI_ZONE"
+	envVenafiUsername       = "VENAFI_USER"
+	envVenafiPassword       = "VENAFI_PASS"
+	envVenafiAccessToken    = "VENAFI_TOKEN"
+	envVenafiApiKey         = "VENAFI_API"
+	envVenafiDevMode        = "VENAFI_DEVMODE"
+	envVenafiP12Certificate = "VENAFI_P12_CERTIFICATE"
+	envVenafiP12Password    = "VENAFI_P12_PASSWORD"
+	envVenafiClientID       = "VENAFI_CLIENT_ID"
+
+	// Attributes of the provider
+	fURL         = "url"
```

Comment:
Could you follow this pattern?

`provider + Atr + <attribute name>`

for this case:

`providerAttrUrl`

Ref: https://github.com/Venafi/terraform-provider-venafi/blob/master/venafi/resource_venafi_certificate.go#L69

Reply:
Ok , my bad. You mean the variable names.

Why do I need to create such long names for the variable? Is there any other provider living on this plugin?
I can agree on changin `fUrl` to `attrUrl`, as attribute is the correct name of this variables in terraform world. But I cannot see how adding the provider to all of them makes the code any better. It is just more unnecessary noise as this plugin is specifically for this provider, no need to differentiate. Context gives you all you need to understand what `fURL` or `attrURL` means

Reply:
renamed constants to `provider<Name>`